qa/L0_http_fuzz/fuzztest.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import unittest
+import sqlite3
+from boofuzz import *
+import glob
+import os
+
+
+class FuzzTest(unittest.TestCase):
+    def _run_fuzz(self):
+        session = Session(target=Target(
+            connection=TCPSocketConnection("127.0.0.1", 8000)),
+            keep_web_open=False)
+
+        s_initialize(name="Request")
+        with s_block("Request-Line"):
+            s_group("Method", ["GET", "HEAD", "POST", "PUT",
+                               "DELETE", "CONNECT", "OPTIONS", "TRACE"])
+            s_delim(" ", name="space-1")
+            s_group("Request-URI", ["/v2/models/simple", "/v2/models/simple/infer",
+                                    "/v2/models/simple/versions/v1", "/v2/models/simple/versions/v1",
+                                    "/v2/models/simple/config", "/v2/models/simple/stats",
+                                    "/v2/health/ready", "/v2/health/live",
```

Comment:
I think server metadata (/v2) and model ready are still missing.

Reply:
Fixed